### PR TITLE
[T5A1][T13-B4] Aloisius Stephen

### PR DESCRIPTION
**Pull request by aloisiusStephen**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | assign persons into a group | search based on the group( e.g. family, colleague, friends)
+`* * *` | user | add multiple phone numbers for a person | save a person's home / work/ handphone contact numbers
```
Comment (zhongwei-z) [inline doc/DeveloperGuide.md]: A very useful feature that I have not seen in others' user stories. :+1:

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -74,12 +77,44 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User enters command to rename tag "A" to "B".
```
Comment (zhongwei-z) [inline doc/DeveloperGuide.md]: Does the user know all the tags? Maybe we can let him/her to list all the tags first.

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -74,12 +77,44 @@ Use case ends.
...
+3. User confirms change.
+4. AddressBook renames tag from "A" to "B" and displays the change.
+Use case ends.
+
+**Extensions**
+1a. User enters command to rename tag without further input
+
+> 1a1. AddressBook prints help message for rename tag command.
+> 1a2. User enters new command.
+> Steps 1a1-1a2 are repeated until the command is recognized.
```
Comment (zhongwei-z) [inline doc/DeveloperGuide.md]: What if the user enters by mistake, or just wants to abort?

File: doc/DeveloperGuide.md [hunk 4]
```diff
@@ -74,12 +77,44 @@ Use case ends.
...
+> Use case Ends
+
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should take less than 3 seconds to execute commands
+6. Should contain straightforward and simple commands
```
Comment (zhongwei-z) [inline doc/DeveloperGuide.md]: Any quantitative metrics of "straightforward" and "simple"?

**Comment by aloisiusStephen** [comment]
Thanks for the comments!

**Pull request closed without merging**